odufrn_downloader/mixins/filters/YearsMixin.py

```diff
@@ -16,8 +16,6 @@ def year_find(self, package_name: str, years: list) -> bool:
         ----------
         bool
             True se o ano foi encontrado no nome do pacote se não false."""
-        if years is None:
```

Comment:
Porque você tirou o caso base?

Reply:
Realmente, teria que ter um bom motivo. Do jeito que tá, acabou mudando o funcionamento da função.

Reply:
Fiz os testes e esse trecho não era utilizado, visto que a função está sendo utilizada em um só local (no download de pacotes): 

```python
if years is None or self.year_find(resource['name'], years):
    self._download(path, resource)
```

Então, se years for None ele faz o download de toda forma. Inclusive isso me fez pensar bastante nesse Mixin, se ele valia a pena existir, por mim seria só uma função privada na classe de pacote.

Reply:
Não tinha notado esse trecho no `download_package`